counsel.el

```diff
@@ -5138,6 +5141,37 @@ The optional `BUILD-ROOT' is useful for other helpers that have found
      local-files)
     compiles))
 
+(defun counsel--find-build-subdir (root)
+  "Return builds sub-directory of `ROOT', if one exists."
+  (cl-some
+   #'(lambda (x)
+       (let ((check (format "%s%s" root x)))
```

Comment:
No need for `/`, you sure?

Reply:
The paths where coming out with trailing slashes but you are right I should be explicit...

Reply:
Just use `expand-file-name`.